chainer_chemistry/datasets/molnet/molnet.py

```diff
@@ -65,21 +67,32 @@ def postprocess_label(label_list):
                            smiles_col=dataset_config['smiles_columns'],
                            postprocess_label=postprocess_label)
     if dataset_config['dataset_type'] == 'one_file_csv':
+        split = dataset_config['split'] if split is None else split
+
+        if isinstance(split, str):
+            splitter = split_method_dict[split]()
+        elif isinstance(split, BaseSplitter):
+            splitter = split
+
+        if isinstance(splitter, ScaffoldSplitter):
+            get_smiles = True
+        else:
+            get_smiles = return_smiles
```

Comment:
I feel adding `else: raise TypeError('message')` is friendly.

Reply:
I think `TypeErro` is not necessary for this line.

Reply:
I want to fix the message to show the "actual" type.